deployment.yaml

```
@@ -89,6 +89,12 @@ parameters:
 - description: memory limit of service
   name: MEMORY_LIMIT
   value: 512Mi
+- description: Cpu request of service
```

```suggestion
- description: CPU request of service
```
Just one minor suggestion.